build/yamls/antrea-aks.yml

```diff
@@ -4373,6 +4394,22 @@ spec:
           readOnly: true
         - mountPath: /var/run/antrea
           name: host-var-run-antrea
+      - command:
+        - install_whereabouts_config
```

Comment:
for some reason, the install_whereabouts_config was not exec during init_container. For now, do manual exec in antrea-agent context before creating pods with Annotation. Will get it addressed shortly.